### PR TITLE
Add tests as a requirement for release.

### PR DESCRIPTION
**Pull request by bdfinst**

### CODE REVIEW

**Review by wirewc — DISMISSED** [review]
Reviewed with @bdfinst, look great!

File: .github/workflows/ci-on-push.yaml
Comment (lucasrod16) [inline .github/workflows/ci-on-push.yaml:1]: I don't think we need a pipeline that only builds the packages. Building the packages in the test workflows should be sufficient for catching package creation errors. This duplicates that process

File: .github/actions/create-bb-package/action.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: Create DUBBED Package
```
Comment (lucasrod16) [inline .github/actions/create-bb-package/action.yaml:1]: `DUBBED` --> `DUBBD`

File: .github/actions/create-bb-package/action.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+description: Creates a DUBBED Zarf package for the requested path
```
Comment (lucasrod16) [inline .github/actions/create-bb-package/action.yaml:2]: `DUBBED` --> `DUBBD`

File: .github/workflows/ci-on-push.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+name: DUBBED package CI
```
Comment (lucasrod16) [inline .github/workflows/ci-on-push.yaml:1]: `DUBBED` --> `DUBBD`

File: .github/workflows/ci-on-push.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+on:
+  push:
+
+jobs:
+  validate-DUBBD:
+    runs-on: ubuntu-latest
+
+    name: Build DUBBED Distro
```
Comment (lucasrod16) [inline .github/workflows/ci-on-push.yaml:9]: `DUBBED` --> `DUBBD`

File: .github/workflows/ci-on-push.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      - uses: ./.github/actions/create-bb-package
+        with:
+          username: ${{ secrets.REGISTRY1_USERNAME }}
+          password: ${{ secrets.REGISTRY1_PASSWORD }}
+          working-dir: defense-unicorns-distro
+
+  validate-DUBBD-AWS:
+    runs-on: ubuntu-latest
+
+    name: Build DUBBED AWS Distro
```
Comment (lucasrod16) [inline .github/workflows/ci-on-push.yaml:24]: `DUBBED` --> `DUBBD`

**Review by lucasrod16 — CHANGES_REQUESTED** [review]

**Comment by lucasrod16** [comment]
We will also need to take a look at changing the name of the required status checks since we are changing the naming on them.

**Comment by bdfinst** [comment]
> We will also need to take a look at changing the name of the required status checks since we are changing the naming on them.

Yes, that's a planned step after we merge this.

**Pull request merged**